.github/workflows/ci.yml

```diff
@@ -334,385 +299,6 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           slack_webhook: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
 
-  build-docker-images:
```

Comment:
question: can we still run this on-demand? for example, if we want to deploy a PR to devnet, we need an image generated for us to be able to do that

Reply:
You will be able to run it on manually on Action thanks to the `workflow_dispatch` that is here on the new workflow created with only docker ilmages

Reply:
awesome, ty!